package.json

```diff
@@ -101,6 +101,11 @@
 						"The documentation view tracks the current cursor position. Display empty content if no documentation is found at the current position.",
 						"The documentation view tries to show the documentation at the current cursor position. If there is none, it continues showing the last available documentation."
 					]
+				},
+				"docsView.documentationView.fontSize": {
+					"type": "integer",
+					"default": 12,
```

Comment:
Should the default be based on the user's editor font size instead?

Reply:
I don't know a nice way to do this. If I use type integer it's not possible to have undefined default. It defaults to 0. 

I could use editor font size if this is 0, but that feels off.

Any ideas on clean ways to do this?